src/integration/integration_scheduler_scalingengine_test.go

```diff
@@ -96,7 +96,7 @@ var _ = Describe("Integration_Scheduler_ScalingEngine", func() {
 
 			Eventually(func() bool {
 				return activeScheduleExists(testAppId)
-			}, 2*time.Minute).Should(BeFalse())
+			}, 2*time.Minute, 5*time.Second).Should(BeFalse())
```

Comment:
out of interest, why 5 seconds rather than 1 second here? seems like some tests changed the interval to 5s, some to 1s.. why? If there's a reason a 1s interval fails here it'd be interesting to understand why? Ideally if we can use 1 second everywhere the tests should be much faster than using 5s.